packages/server/database/migrations/20191111140628-meetingNames.ts

```diff
@@ -7,7 +7,7 @@ export const up = async function(r: R) {
       name: r.branch(
         row('meetingType').eq('retrospective'),
         r('Retro #').add(row('meetingNumber').coerceTo('string')),
-        r('Action meeting #').add(row('meetingNumber').coerceTo('string'))
+        r('Check-in meeting #').add(row('meetingNumber').coerceTo('string'))
```

Comment:
@mattkrick I checked in this commit so that I could ping you. I’m guessing it’s not necessary to change the name of past meetings, and even if we wanted to, we’d probably have to write a new migration? I can revert this, but was curious.

Reply:
yep, rule of thumb is if a migration has hit prod, we can't touch it or bad things happen.
we'll probably want to rename previous meetings by looking for the phrase "Action meeting" and replacing it.